index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+
+
+
+.. _intro:
+
+Introduction
+============
+
+This document describes the project planning processes and tools used in the DM,
+with particular emphasis on software development.
+
+LSST is now in construction, funded through NSF MREFC. The planning process has
+to conform to all government standards and requirements, including mandated
+Risk and Earned Value Management System (EVMS).
+
+The team responsible for delivering the LSST DM Software is approaching 100 engineers and
+scientists, and is highly distributed geographically. It is essential all processes are robust
+and clearly defined, and everyone understands how to apply them; what tools to use and how.
+
+The DM Software is large and complex, and will be written over the entire period of
+the construction (7 years). To ensure the team delivers production-quality software
+that meets baseline specification on time and within budget, having a solid, realistic,
+resource-loaded, well maintained long-term plan is a must. Further,
+appropriate tools and processes need to be in place to track the progress and
+execute the long-term plan.
+
+Using modern techniques, in particular the Agile Process is extremely useful in particular
+for short-term planning, managing short-term activities, and maximizing
+team's efficiency.
+
+The project planning processes described in this documents have been structured to
+address all of the above (often not fully aligned!) needs. It is a fusion of
+the Earned Value Management System, and the Project Planning with Agile Process.
+
+
+.. _baseline-plan:
+
+DM Project Management Planning Process
+======================================
+
+This section explains how the DM Subsystem approaches project planning and plan execution.
+
+Baseline Documents
+------------------
+
+The baseline design and architecture of the DM Subsystem is captured through a set of change-controlled
+documents, including:
+
+* LDM-148 DM System Design
+
+* LDM-151 DM Applications Design
+
+* LDM-152 DM Middleware Design
+
+* LDM-135 DM Database Design
+
+* LDM-131 DM SUIT Conceptual Design
+
+The complete set of DM baseline documents can be found in Docushare, Collection-2511.
+
+All together, this collection of documents capture all work DM has to deliver during the LSST Construction.
+
+Planning Packages
+-----------------
+The work is divided into *o(few hundred)* concrete, technical *packages of planned work*. The packages divide work
+at the 3rd Level of WBS, e.g., typically a 3rd Level WBS is broken down into 20-50 packages. Each such package
+addresses a piece of work defined in the baseline, and all together they cover the entire DM Baseline.
+Each package has a clearly defined scope, deliverable(s), resource cost, and an end date. Additionally,
+each package keeps track of which section(s) of which baseline document it implements.
+
+
+
+In the EVMS world, these packages are implemented directly as *Planning Packages* (PP).
```

Comment:
I believe it is attempting to state there is a 1 to 1 relationship between the Collection-2511 planning packages and the PMCS planning packages.